src/lib.rs

```diff
@@ -20,6 +20,45 @@ mod fetch;
 mod manifest;
 mod dependency;
 
+mod errors {
+    error_chain!{
+        errors {
+            /// Failed to fetch crate from crates.io
+            FetchVersionFailure {
+                description("Failed to fetch crate version from crates.io")
+            }
+            /// Invalid JSON from crates.io response
+            InvalidCratesIoJson { description("Invalid JSON (the crate may not exist)")}
+            /// No versions available
+            NoVersionsAvailable {
+                description("No available versions exist. Either all were yanked\
+                             or only prerelease versions exist. Trying with the\
```

Comment:
I think this leaves no space between the words in these to lines, you'll need to write it like `yanked \`